docs/ruleset_functions/zone_meets_G3_1_1e.md

```diff
@@ -0,0 +1,60 @@
+# zone_meets_G3_1_1e

```

Comment:
Could we change this function by adding does as prefix?
`does_zone_meets_G3_1_1e`? The name could indicate this function is returns boolean